### ISSUE TITLE
replace with matrix-org/matrix-appservice-slack

### ISSUE THREAD

**Issue by nikhiljha** [ocf/slackbridge#107]
https://github.com/matrix-org/matrix-appservice-slack

https://github.com/ocf/projects/issues/52

**Comment by jvperrin** [comment]
Bold of you to assume other software has no bugs and fits all our use cases

**Comment by jvperrin** [comment]
(to be clear, I don't mind this project getting replaced by a better-maintained alternative, but I disagree that this will "fix bugs 1 through 106" and in saying that you're ignoring that around half of those were PRs that people put their time and effort into)

**Comment by nikhiljha** [comment]
("fixes bugs 1 through 106" is satirical)

**Comment by jvperrin** [comment]
Fair enough, I figured it probably was, but it's hard to tell over text and I think even with satire there's a kernel of truth to it. I overreacted because this is a project I've put a lot of time and effort into.

I do think if this replacement goes well it'll fix some bugs with the bridging, or at least make it more standard, which is good news. I'm interested to see how it goes and what format messages, attachments, etc. get bridged through as.
